### DISCUSSION TITLE
Did react-query take inspiration from DigitalGlobe/jetset?

### DISCUSSION THREAD

**Discussion by glortho** [TanStack/query#2603] [Q&A] [answered]
👋🏼 Greetings! 

Some colleagues introduced me to React Query recently and first of all, it's awesome ✨. 

Second of all, I was struck by how similar it is in terms of some core functionality and interfaces to https://github.com/DigitalGlobe/jetset (of which I was primary author 😄), which was born 4 years ago and mostly went unmaintained starting 2 years ago.

Lo and behold, I found @tannerlinsley there as one of jetset's [stargazers](https://github.com/DigitalGlobe/jetset/stargazers)!

<img width="715" alt="Screen Shot 2021-08-26 at 14 58 10" src="https://user-images.githubusercontent.com/526284/131022008-7f84d4b4-d94f-4fe3-9587-6cfac363422a.png">

I don't see any reference to jetset in this repo, so maybe the author doesn't trace any lineage of ideas across these projects. But if that's just an oversight, some attribution would be a nice gesture given the illustrious status that React Query currently enjoys. 😄 

Thank you for all your excellent work here and across the ecosystem! 💖

**Comment by tannerlinsley** [comment, marked as answer]
I am so sorry to say that it wasn’t an original inspiration :( Thanks for that goes to Draqula and Apollo, but I do remember the name and star gazing it. Regardless it looks like a cool lib!

Tanner Linsley

**Reply by glortho** [reply]
Fair enough! Thanks for the response. ✨
